### PR TITLE
[HTV4-73] Dashboard navigation

### PR DESCRIPTION
**Pull request by leungbyr**
Desktop
image
Mobile
image

### CODE REVIEW

File: src/components/Navigation.jsx [hunk 1]
```diff
@@ -1,18 +1,158 @@
...
+
+    ${Mixins.media(
+        'tablet',
+        `
+        padding: 15px 0 15px 0;
+    `,
+    )}
+`;
+
+const LogoutLink = styled(L)`
```
Comment (fpunny) [inline src/components/Navigation.jsx]: might be better to make it extend from NavLink to make it fit the look better (aka fix the weird non-centering of the logout icon)

File: src/components/Navigation.jsx [hunk 2]
```diff
@@ -1,18 +1,158 @@
...
+    ${Mixins.media(
+        'tablet',
+        `
+        padding: 15px 0 15px 0;
+    `,
+    )}
+`;
+
+const MailLink = styled.a`
+    display: flex;
```
Comment (fpunny) [inline src/components/Navigation.jsx]: The styles for this and NavLink can be abstracted to make it more reuseable

File: src/components/Navigation.jsx [hunk 3]
```diff
@@ -1,18 +1,158 @@
...
+                            <Icon as={routeProps.Icon} />
+                            <Text>{key}</Text>
+                        </NavLink>
                     </li>
                 ))}
-                <li>Logout</li>
-            </ul>
-        </nav>
+                <li>
+                    <MailLink
```
Comment (fpunny) [inline src/components/Navigation.jsx]: Rather than do this, use NavLink and use `as`

File: src/layouts/Dashboard.jsx [hunk 4]
```diff
@@ -6,7 +6,7 @@ const routes = Object.entries(pages);
 
 export const Dashboard = () => {
     return (
-        <main>
+        <main style={{ height: '100%' }}>
```
Comment (fpunny) [inline src/layouts/Dashboard.jsx]: pls no, use a styled component

**Comment by fpunny** [comment]
The text should not be hidden behind the nav btw

File: src/components/Navigation.jsx [hunk 5]
```diff
@@ -1,18 +1,158 @@
...
+        </Nav>
     );
 };
+
+const Nav = styled.nav`
+    height: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 220px;
```
Comment (fpunny) [inline src/components/Navigation.jsx]: put this in themes

File: src/components/Navigation.jsx [hunk 6]
```diff
@@ -1,18 +1,158 @@
...
+    ${Mixins.media(
+        'tablet',
+        `
+        padding: 15px 0 15px 0;
+    `,
+    )}
+`;
+
+const LogoutLink = styled(L)`
+    position: absolute;
```
Comment (fpunny) [inline src/components/Navigation.jsx]: i dont think this should be required (I would stay away from breaking flow)

**Comment by leungbyr** [comment]
Desktop
image
Mobile
image

**Comment by fpunny** [comment]
Looks good enough for now, we can tweek this in another iteration

**Pull request merged**